cmds/fmap/fmap.go

```diff
@@ -37,10 +41,12 @@ import (
 )
 
 var (
-	checksum = flag.String("c", "", "print checksum using the given `hash function` (md5|sha1|sha256)")
-	read     = flag.Int("r", -1, "read an area from the flash")
-	summary  = flag.Bool("s", false, "print human readable summary")
-	usage    = flag.Bool("u", false, "print human readable usage summary")
+	checksum  = flag.String("c", "", "print checksum using the given `hash function` (md5|sha1|sha256)")
+	read      = flag.Int("r", -1, "read an area from the flash")
```

Comment:
I think it's time to consider skipping the flags and go with a verb. That would simplify your error checking below. command = map[string]something {"read": whatever}, etc.}